src/modding/ModManager.cs

```diff
@@ -509,7 +743,14 @@ private void RefreshEnabledMods()
 
         foreach (var mod in enabledMods)
         {
-            enabledModsContainer.AddItem(mod.InternalName, LoadModIcon(mod));
+            if (mod != null)
```

Comment:
You should instead fix things so that no null mods can be in the enabledMods list rather than checking things here.